CacheService/docker_context/CacheHandler.py

```diff
@@ -47,6 +48,19 @@ async def getUserFunds(self, user_id):
             else:
                 return result, status
 
+    async def getUser(self, user_id):
```

Comment:
Could we make the getUserFunds method above use this to clean up the logic a bit?